docs/usage_guide.rst

```diff
@@ -593,25 +593,45 @@ The list of currently supported airdrops is:
 Manually adding trades or other events
 **************************************
 
+Adding historical events
```

Comment:
No no. There is already a title for the section above.

Change 

> Manually adding trades or other events

to

Adding historical events.

And your "historical events"

to 

manual trades.

It should be obvious if you do a `make html` and go and read the generated docs locally.